### ISSUE TITLE
Q1Results is not updated

### ISSUE THREAD

**Issue by serezhub** [alex-wolf-ps/csharp-program-flow#3]
It is missing the `AreaToImprove` get/set and the individual responses.  I was able to add the AreaToImprove part to continue with the course, but the individual responses are necessary for the loop section.

**Comment by Eurodrome** [comment]
Am i missing something here? In the demo we speak of a new List<SurveyResponse>(), but i never see what's in that file (SurveyResponse.cs) so i can't get further.

**Comment by serezhub** [comment]
Yeah...not having the data that he uses in the videos makes it impossible to implement the loops exercises ourselves in Visual Studio.  I found this course for free on Udemy and it's excellent https://www.udemy.com/course/understandingc/learn/lecture/2216938#overview

**Comment by Eurodrome** [comment]
I found out that the code is complete @ PluralSight. When you go to the details of this course, you can download the zip where per course a different folder is stored.
